resources/lang/de_DE/default.po

```diff
@@ -183,26 +186,26 @@ msgstr "Passwort vergessen"
 
 #: resources/views/pages/login.twig:85
 msgid "Please sign up, if you want to help us!"
-msgstr "Bitte registriere Dich, wenn Du helfen möchtest!"
+msgstr "Bitte registriere Dich, wenn Du Arbeitsplätze buchen möchtest!"
 
 #: resources/views/pages/login.twig:90 includes/pages/guest_login.php:61
 msgid "Registration is disabled."
 msgstr "Registrierung ist abgeschaltet."
 
 #: resources/views/pages/login.twig:95
 msgid "What can I do?"
-msgstr "Was kann ich machen?"
+msgstr "Welche Arbeitsplätze kann ich buchen?"
 
 #: resources/views/pages/login.twig:96
 msgid "Please read about the jobs you can do to help us."
 msgstr ""
-"Bitte informiere Dich über die Tätigkeiten bei denen Du uns helfen kannst."
+"Das hängt von Deinen Engeltypen ab."
 
 #: resources/views/pages/login.twig:99
 #: includes/controller/angeltypes_controller.php:69
 #: includes/view/AngelTypes_view.php:501 includes/view/AngelTypes_view.php:598
 msgid "Teams/Job description"
-msgstr "Team-/Aufgabenbeschreibung"
+msgstr "Beschreibung der Engeltypen"
```

Comment:
nope

Reply:
Of course this is not the correct translation of `Teams/Job description`, but `Teams/Job description` is just the message ID. In the place where this message appeared the string `Description of angeltypes` is much more fitting, so I changed both the german and english translation. See arbeitsplatz.stuvus.de for a demo.

Reply:
Also see https://github.com/stuvusIT/handtuchsystem/pull/2/commits/ab920c275789cc8339252b6ddd797f73b7571a39#diff-3dda712beb445dc926265a69e01c2248R208